haddock-heroku.rb

```diff
@@ -29,3 +29,13 @@
   10.times{ @passwords << Haddock::Password.generate(@length) }
   erb :index
 end
+
+get '/:len'
+ @length = case
```

Comment:
Shouldn't you need `.to_i` in order to cast to an int? Also we should probably handle the non-int input cases by returning a message that indicates integer values in a specific range should be used.